apps/elewa-group-website/src/app/libs/elements/layout/elewa-hero/elewa-hero.component.scss

```diff
@@ -0,0 +1,92 @@
+*,
+*::after,
+*::before{
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+.root {
+    background-size: cover;
+    height: 100vh;
+    padding: 0;
+    margin: 0;
+    width: auto;
+}
+
+li, a{
+    font-size: 16px;
+}
+
+.container{
+    padding: 30px 10%;
+    display: block;
+}
+.navbar-container{
+    display: flex;
+    justify-content: space-between;
+    height: 50px;
+    align-items: center;
+    border-bottom: 2px solid white;
+}
+
+.menu-items{
+    display: flex;
+}
+
+.menu-items li{
+    list-style: none;
+    margin-left: 1.5rem;
+    font-size: 1.3rem;
+}
+
+.navbar a{
+    color: #444;
```

Comment:
Needs to use SCSS variables + navbar is not part of scope of this issue so please remove

Reply:
resolved